PyISY/Nodes/__init__.py

```diff
@@ -187,8 +187,22 @@ def parse(self, xml):
                                 dimmable = '%' in \
                                     nprop[0].attributes['uom'].value
                                 nval = int(nval.replace(' ', '0'))
+                            # Get the device notes, currently only the Spoken property is saved.
+                            notes_xml = self.parent.conn.getNodeNotes(nid)
```

Comment:
Makes sense, I'll do that tomorrow.

On Wed, Sep 2, 2015, 9:25 PM Ryan Kraus <notifications@github.com> wrote:

> In PyISY/Nodes/__init__.py
> <https://github.com/automicus/PyISY/pull/10#discussion_r38612208>:
>
> > @@ -187,8 +187,22 @@ def parse(self, xml):
> >                                  dimmable = '%' in \
> >                                      nprop[0].attributes['uom'].value
> >                                  nval = int(nval.replace(' ', '0'))
> > +                            # Get the device notes, currently only the Spoken property is saved.
> > +                            notes_xml = self.parent.conn.getNodeNotes(nid)
>
> Pretty much. I think that would be a better option because this data is
> not sent in the bulk XML data and it is somewhat ancillary.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/automicus/PyISY/pull/10/files#r38612208>.
>